2024-11-19T16:48:23.653886+0000 INFO 139845321175168 Created database engine for db quotesdbtest_whistleask, user quotesdbtest_whistleask, address pcyp4.h.filess.io:3307
2024-11-19T16:48:24.393643+0000 INFO 139845321175168 Database schema initialized successfully.
2024-11-19T16:48:24.504210+0000 INFO 139845321175168 Tables in the database: ['TestTable', 'authors', 'quotes', 'quotes_tags_link', 'tags', 'users']
2024-11-19T16:48:24.504616+0000 INFO 139845321175168 All required tables have been created successfully.
2024-11-19T16:48:24.790100+0000 INFO 139845321175168 All tables truncated successfully.
2024-11-19T16:48:24.851658+0000 INFO 139845321175168 Database initialized successfully with new records.
2024-11-19T16:49:50.084461+0000 ERROR 139845321175168 Error executing statement 'SELECT users.id, users.username, users.password 
FROM users 
WHERE users.username = :username_1': (pymysql.err.OperationalError) (1054, "Unknown column 'users.id' in 'field list'")
[SQL: SELECT users.id, users.username, users.password 
FROM users 
WHERE users.username = %(username_1)s]
[parameters: {'username_1': 'user3'}]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
